README.md

```diff
@@ -103,7 +103,7 @@ Sequential(
 
 ### Example input
 
-Test data - this file should contain a column with the feature names (quantmatrix or some matrix containing input column - in this case "Protein")
+**Data** - this file should contain a column with the feature names (quantmatrix or some matrix containing input column - in this case "Protein"). These *need* to map to the input layer of the BINN, either directly or by providing a translation file. 
```

Comment:
Maybe we should provide a general example where we do not use a translation file to explain the input in the most basic terms?